src/redis-cli.c

```diff
@@ -1978,6 +1993,7 @@ static void usage(int err) {
 "  --quoted-input     Force input to be handled as quoted strings.\n"
 "  --csv              Output in CSV format.\n"
 "  --json             Output in JSON format (default RESP3, use -2 if you want to use with RESP2).\n"
+"  --json-binary-encoding Output JSON value as escaped binary character with \\x?? style\n"
```

Comment:
Truth be told, I don't really understand the value of this. 
Maybe improve the doc, and please also update the PR top comment to list that option and explain it.

Reply:
Updated the PR description.